paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/SavedPaymentMethodTab.kt

```diff
@@ -105,6 +107,13 @@ internal fun SavedPaymentMethodTab(
                             contentDescription = description.readNumbersAsIndividualDigits()
                         }
                 )
+
+                if (shouldShowDefaultBadge && editState != PaymentOptionEditState.None) {
```

Comment:
why is edit state relevant here? Edit state represents whether the PM can be edited, I would expect that we would want to show the default badge even if the PM could not be edited

Reply:
As per figma, its not supposed to show the default badge when its not being edited.

https://www.figma.com/design/ysEUcRWOfdEH4C0jBA0EkH/MPE-LPM-Visibility-%26-Vertical-Mode-Rollout?node-id=9285-197341&t=LieDNDwpzpqnZpUF-4

Reply:
ah, interesting. Thanks for catching that! Can you make sure to add a test for this behavior? Feels unintuitive to me and I think could be easy to regress in the future